README.md

```diff
@@ -1,13 +1,13 @@
 # WaveRL
-A package for training RL agents to perform active damping on a model of a vibrating bridge.
+A package for training RL agents to perform active damping on a model of a oscillating bridge.
 
 ## Contents
 * complex_envs/ :  This folder contains the code for the environments as well as scripts for training and rolling out agents
```

Comment:
should be organized and described according to coding standards

Reply:
Is this a reiteration of the previous comment?  I will be fixing the organization of the files but do you have suggestions or more specific critique of the descriptions?